rct229/rules/section5.py

```diff
@@ -0,0 +1,58 @@
+from numpy import sum
+
+from rct229.data.schema_enums import schema_enums
+from rct229.rule_engine.rule_base import (
+    RuleDefinitionBase,
+    RuleDefinitionListIndexedBase,
+)
+from rct229.rule_engine.user_baseline_proposed_vals import UserBaselineProposedVals
+from rct229.utils.jsonpath_utils import find_all
+
+# Rule Definitions for Section 5 of 90.1-2019 Appendix G
+
+
+# ------------------------
+
+
+class Section5Rule3(RuleDefinitionListIndexedBase):
+    """Rule 3 of ASHRAE 90.1-2019 Appendix G Section 6 (Lighting)"""
+
+    def __init__(self):
```

Comment:
I found descriptions of table functions really helpful. Would it make sense to include similar documentation for classes/functions and their return types within sections as well.?

Reply:
The documentation can be found in rule_base.py related to sections.